packages/nextjs/src/server/clerkClient.ts

```diff
@@ -71,16 +71,14 @@ const clerkClientForRequest = () => {
 };
 
 interface ClerkClientExport extends ClerkClient {
-  (): ClerkClient;
+  (): Promise<ClerkClient>;
 }
 
 // TODO SDK-1839 - Remove `clerkClient` singleton in the next major version of `@clerk/nextjs`
-const clerkClient = new Proxy(Object.assign(clerkClientForRequest, clerkClientSingleton), {
+export const clerkClient = new Proxy(Object.assign(clerkClientForRequest, clerkClientSingleton), {
```

Comment:
@BRKalow With all of breaking changes in this PR, should we follow through on this removal also?

Reply:
Looks like yes: https://github.com/clerk/javascript/pull/4356

Reply:
Ooh good call @jacekradko, yes!